### PR TITLE
User status + fix sagas logic

### PR DESCRIPTION
**Pull request by montakarlo**

### CODE REVIEW

File: frontend/src/common/helpers/apiHelper.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { stringifyUrl, ParsedQuery } from 'query-string';
 import { FetchMethod } from '../enums/FetchMethod';
-import { env } from 'env';
+import { env } from '../../env';
```
Comment (vlvereta) [inline frontend/src/common/helpers/apiHelper.ts:3]: Fix import routes

File: frontend/src/common/models/message/ISendMessage.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export interface ISendMessage {
+  message: string;
+}
```
Comment (vlvereta) [inline frontend/src/common/models/message/ISendMessage.ts:3]: What's it?

File: frontend/src/common/models/user/IUser.ts [hunk 3]
```diff
@@ -5,4 +5,5 @@ export interface IUser {
   displayName: string;
   imageUrl?: string | null;
   title?: string | null;
+  status?: string | null;
```
Comment (vlvereta) [inline frontend/src/common/models/user/IUser.ts:8]: We can skip null in interfaces, check please.

File: frontend/src/components/CreateChannelForm/index.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import { Button, Form, InputGroup } from 'react-bootstrap';
-import { IBindingCallback1 } from 'common/models/callback/IBindingCallback1';
-import { ICreateChannel } from 'common/models/channel/ICreateChannel';
+import { IBindingCallback1 } from '../../common/models/callback/IBindingCallback1';
+import { ICreateChannel } from '../../common/models/channel/ICreateChannel';
```
Comment (vlvereta) [inline frontend/src/components/CreateChannelForm/index.tsx:4]: Fix imports.

File: frontend/src/components/EmojiPopUp/styles.module.sass [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.popUpContainer
+  background-color: transparent
+  border: none
+  padding: 0px
+  box-sizing: border-box
+  max-width: 280px
+
+.emojiContainer>aside
+  box-shadow: none
+  transition: none
```
Comment (vlvereta) [inline frontend/src/components/EmojiPopUp/styles.module.sass:10]: Fix.

File: frontend/src/components/ProfilePreview/index.tsx [hunk 6]
```diff
@@ -0,0 +1,96 @@
+import styles from './styles.module.sass';
```
Comment (vlvereta) [inline frontend/src/components/ProfilePreview/index.tsx:1]: Why this file is here?

File: frontend/src/components/TextEditor/index.tsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import styles from './styles.module.sass';
 import React, { FunctionComponent } from 'react';
 import { Editor } from '@tinymce/tinymce-react';
-import { env } from 'env';
+import { env } from '../../env';
```
Comment (vlvereta) [inline frontend/src/components/TextEditor/index.tsx:4]: Fix imports.

File: frontend/src/services/userService.ts [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  const response = editStatusData.status; // mocked
+  return response;
+}
+
+export async function deleteUser() {
+  await api.delete('/api/users/');
+}
+
+export async function editUser(data: IUser) {
+  const response = await api.put<IUser>('/api/users/', { data });
```
Comment (AlikRakhmonov) [inline frontend/src/services/userService.ts:21]:
```suggestion
  const response = await api.put<IUser>('/api/users', { data });
```

you don't need slash in the end, remove it everywhere

File: frontend/src/sagas/user.ts [hunk 9]
```diff
@@ -18,16 +19,23 @@ import { Routine } from 'redux-saga-routines';
...
+import { IUser } from 'common/models/user/IUser';
+import { editStatus, deleteUser, editUser, forgotPassword, resetPassword } from 'services/userService';
+<<<<<<< HEAD
+import { history } from 'common/helpers/historyHelper';
+import { toastrError } from 'services/toastrService';
+=======
+import { toastrError } from 'services/toastrService'
+import { history } from 'common/helpers/historyHelper';
 import { push } from 'connected-react-router';
+>>>>>>> dev
```
Comment (AlikRakhmonov) [inline frontend/src/sagas/user.ts:31]: invalid merge

**Pull request merged**